molecule/oauth-plain-rhel/verify.yml

```diff
@@ -193,27 +193,39 @@
       vars:
         file_path: /etc/kafka/connect-distributed.properties
         property: security.protocol
-        expected_value: SASL_PLAINTEXT
+        expected_value: SASL_SSL
 
     - import_role:
         name: confluent.test
         tasks_from: check_property.yml
       vars:
         file_path: /etc/kafka/connect-distributed.properties
         property: producer.sasl.jaas.config
-        expected_value: 'org.apache.kafka.common.security.plain.PlainLoginModule required username="kafka_connect" password="kafka_connect-secret";'
+        expected_value: 'org.apache.kafka.common.security.oauthbearer.OAuthBearerLoginModule required clientId="kafka_connect" clientSecret="my-secret" ssl.truststore.location="/var/ssl/private/kafka_connect.truststore.jks" ssl.truststore.password="confluenttruststorepass";'
 
     - name: Check Updated Service Description
       shell: systemctl status confluent-kafka-connect
       register: systemctl_status
       failed_when: "'Custom Connect description' not in systemctl_status.stdout_lines[0]"
       changed_when: false
 
+    - import_role:
+        name: confluent.platform.common
+        tasks_from: get_authorization_token.yml
+      vars:
+        oauth: "{{ kafka_connect_oauth_enabled }}"
+        oauth_user: "{{ kafka_connect_oauth_user }}"
+        oauth_password: "{{ kafka_connect_oauth_password }}"
+        ldap_user: "{{ mds_super_user }}"
```

Comment:
shouldn't ldap user be kafka_connect_ldap_user?

Reply:
Yes will update it